cmd/cli/app/root.go

```diff
@@ -82,8 +82,9 @@ func initConfig() {
 	viper.SetEnvPrefix("rekor")
 	viper.AutomaticEnv()
 
-	if err := viper.ReadInConfig(); err == nil {
-		fmt.Println("Using config file:", viper.ConfigFileUsed())
+	if err := viper.ReadInConfig(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
It looks like this one might be breaking the tests.

Reply:
yeah, you're right.